packages/schema/src/doc.ts

```diff
@@ -123,11 +147,9 @@ export class JupyterGISDoc
   }
 
   updateLayer(id: string, value: IJGISLayer): void {
-    const obj = this._getLayerAsYMap(id);
-    if (!obj) {
-      return;
-    }
-    this.transact(() => obj.set(id, value));
+    this.transact(() => {
+      this._layers.set(id, value);
+    })
```

Comment:
This PR changes the way the layers are updated in the shared document.

The previous implementation raised errors like `obj.set is undefined`

Reply:
Thanks! This was a wrong copy from jupytercad where we have a **list** of objects, here we have a **dictionary**